### PR TITLE
[Reference cleanup]LPS-161154

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references in document-library-service. Thanks!
https://issues.liferay.com/browse/LPS-161154

### CODE REVIEW

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/DLFileEntryTypeStagedModelDataHandler.java [hunk 1]
```diff
@@ -454,12 +428,21 @@ private DLFileEntryType _fetchExistingFileEntryTypeWithParentGroups(
...
 	private DDMStructureLocalService _ddmStructureLocalService;
+
+	@Reference
 	private DLFileEntryTypeLocalService _dlFileEntryTypeLocalService;
 
 	@Reference
 	private GroupLocalService _groupLocalService;
 
+	@Reference(
+		target = "(&(verify.process.name=com.liferay.document.library.service))"
```
Comment (dantewang) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/DLFileEntryTypeStagedModelDataHandler.java:441]: @JanisZhang Could you check whether `target = "(verify.process.name=com.liferay.document.library.service)"` works?

Basically, when using `&`, we usually write `(&(A)(B))`, meaning `A && B`; if there's only 1 condition, I don't think `&` is needed.

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/FileEntryStagedModelDataHandler.java [hunk 2]
```diff
@@ -1308,6 +1301,11 @@ private void _updateFriendlyURLEntries(
 	@Reference(target = "(ddm.form.values.serializer.type=json)")
 	private DDMFormValuesSerializer _jsonDDMFormValuesSerializer;
 
+	@Reference(
+		target = "(&(verify.process.name=com.liferay.document.library.service))"
```
Comment (dantewang) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/FileEntryStagedModelDataHandler.java:1305]: @JanisZhang Same here.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by zhangyan0701** [comment]
ci:test:document

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by tinatian** [comment]
@JanisZhang Please do the reference change first, then modify the bad filters. Please resend.

Thanks.
Tina

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/DLFileEntryTypeStagedModelDataHandler.java (hunk 1 above)
Comment (tinatian) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/DLFileEntryTypeStagedModelDataHandler.java:441]: @JanisZhang Bad filter here

**Pull request closed without merging**